php/oop/registry/src/Registry/LegacyRegisterOffice.php

```diff
@@ -28,9 +29,14 @@ public function register(Person $person): PersonalIdentifier
 
     public function changeName(PersonalIdentifier $id, string $newName): void
     {
+		   if (isset($this->changedNamedPeople[$id->value()]) && $this->changedNamedPeople[$id->value()] == true){
```

Comment:
Is there any reason to check the second condition? It seems the first condition is enough. You may wanna use [in_array](https://www.php.net/manual/en/function.in-array.php) instead of associative array